README.md

```diff
@@ -17,7 +17,7 @@ There are multiple demos in this repo that showcase Kafka stream processing on t
 | [ksql-workshop](ksql-workshop/README.md)   |   Y   |   Y    | showcases Kafka stream processing using KSQL and can be run automated or self-guided as a KSQL workshop
 | [microservices-orders](microservices-orders/README.md) |   Y   |   N    | integrates the [Microservices Orders Demo Application](https://github.com/confluentinc/kafka-streams-examples/tree/5.0.x/src/main/java/io/confluent/examples/streams/microservices) into Confluent Platform
 | [music](music/README.md)                   |   Y   |   Y    | KSQL version of the [Kafka Streams Demo Application](https://docs.confluent.io/current/streams/kafka-streams-examples/docs/index.html)
-| [mysql-debezium](mysql-debezium/README.md) |   Y   |   N    | end-to-end streaming ETL with KSQL for stream processing using the [Debezium Connector for MySQL](http://debezium.io/docs/connectors/mysql/)
+| [mysql-debezium](mysql-debezium/README.md) |   Y   |   [Here](https://github.com/confluentinc/demo-scene/tree/master/mysql-debezium-ksql-elasticsearch)    | end-to-end streaming ETL with KSQL for stream processing using the [Debezium Connector for MySQL](http://debezium.io/docs/connectors/mysql/)
```

Comment:
Suggestion: change `Here` to `Y`

Reply:
Suggestion: do the same for the other demos that have links elsewhere (e.g. "clickstream": change Docker `N` to `Y` with hyperlink to ksql GH repo; and do the same for "wikipedia" with link to `cp-demo` GH repo)